### PR TITLE
bind 9.16.22

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Comment by chenrui333** [comment]
```
bind:
  * Libraries were compiled with a flat namespace.
    This can cause linker errors due to name collisions, and
    is often due to a bug in detecting the macOS version.
      /opt/homebrew/Cellar/bind/9.16.22/lib/named/filter-aaaa.so
```

**Comment by carlocab** [comment]
Try this:
```diff
diff --git a/configure b/configure
index a56ea16..c646af8 100755
--- a/configure
+++ b/configure
@@ -13628,7 +13628,7 @@ $as_echo "$as_me: WARNING: When using GNU C Compiler on Solaris, -zrelax=transtl
 fi
 	 ;; #(
   *-darwin*) :
-    LDFLAGS="$LDFLAGS -Wl,-flat_namespace" ;; #(
+    LDFLAGS="$LDFLAGS" ;; #(
   *) :
      ;;
 esac
diff --git a/configure.ac b/configure.ac
index e0b4d79..2626f9b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -382,8 +382,7 @@ AS_CASE([$host],
 		  [LDFLAGS="$LDFLAGS -zrelax=transtls"
 		   AC_MSG_WARN([When using GNU C Compiler on Solaris, -zrelax=transtls linker flag is used to fix bug in Thread Local Storage])
 		  ])
-	],
-	[*-darwin*],[LDFLAGS="$LDFLAGS -Wl,-flat_namespace"])
+	])
 
 #
 # CCNOOPT defaults to -O0 on gcc and disables optimization when is last
```
You only need to apply the patch to `configure`. The `configure.ac` can be upstreamed, though that might need cleaning up.

File: Formula/bind.rb [hunk 1]
```diff
@@ -41,6 +41,9 @@ class Bind < Formula
     sha256 "00c7c1aaa88358b9c765b6d3000c6eec0ba42abca5351b095321aef446081da3"
   end
 
+  # Remove use of -flat_namespace
```
Comment (carlocab) [inline Formula/bind.rb:44]: We need to upstream this patch if we're using it. The usual patches we use are to fix bugs in Libtool, and that's already been upstreamed.

Comment (chenrui333) [reply Formula/bind.rb:44]: noted

**Pull request closed without merging**